msal/tests/dev apps/XForms/XForms.Android/MainActivity.cs

```diff
@@ -47,13 +47,13 @@ protected override void OnCreate(Bundle bundle)
             global::Xamarin.Forms.Forms.Init(this, bundle);
             LoadApplication(new App());
 
-            App.UIParent = new UIParent(this);
+            //App.UIParent = new UIParent(this);
```

Comment:
is this dead code?  if so, remove.

Reply:
it is not, we uncomment it when we want to enable the android app to use system webview

Reply:
Could you please add a comment above this to specify the intention then?